src/privileged/fxa/api.js

```diff
@@ -32,21 +32,27 @@ function sanitizeUser(user) {
       avatarDefault = user.profileCache.profile.avatarDefault;
     }
 
-    return {
-      avatar,
-      avatarDefault,
-      email,
-      hashedUid: hashedUid(),
-      verified,
-    };
+    return getHashedUid()
+      .then((hashedUid) => {
```

Comment:
Seeing as we're using `async`/`await` everywhere else, should we prefer it here too?